assets/src/edit-story/components/panels/textStyle/font.js

```diff
@@ -67,30 +104,31 @@ function FontControls({ selectedElements, pushUpdate }) {
             options={fonts}
             value={fontFamily}
             onChange={(value) => {
-              const currentFontWeights = getFontWeight(value);
-              const currentFontFallback = getFontFallback(value);
-              const fontWeightsArr = currentFontWeights.map(
-                ({ value: weight }) => weight
-              );
-
-              // Find the nearest font weight from the available font weight list
-              // If no fontweightsArr available then will return undefined
-              const newFontWeight =
-                fontWeightsArr &&
-                fontWeightsArr.reduce((a, b) =>
-                  Math.abs(parseInt(b) - fontWeight) <
-                  Math.abs(parseInt(a) - fontWeight)
-                    ? b
-                    : a
+              loadFont(value, () => {
```

Comment:
One nuance we need to consider is that `loadFont` needs to be called for a lot more than just family. Including content. So, I'm not sure this is a good place. Maybe it'd be better to have this code in the presubmit callback in the parent component?